src/screens/Assets/utils.js

```diff
@@ -50,11 +46,11 @@ export function buildAssetDataNavigationParam(asset: Asset, { accountAddress }:
     id: symbol,
     name: name || symbol,
     token: symbol,
-    address: accountAddress,
```

Comment:
is it better if we rename this to `accountAddress`? I feel that the term `address` itself is a bit ambiguous in this context...

Reply:
It's not needed here at all, I guess it appeared here by mistake because 🤷